Add "What's new in Astro - October" blogpost (#859)
withastro/astro.build: src/content/blog/whats-new-october.mdx

---
title: "What's new in Astro - October"
description: "Another busy month in Astro! From Hacktoberfest to releasing Page Partials. Read up on everything that happened this month in the Astro universe."
publishDate: "October 31, 2023"
authors:
  - elian
coverImage: "/src/content/blog/_images/whats-new-october/head.webp"
socialImage: "/src/content/blog/_images/whats-new-october/og.webp"
lang: "en"
---

> ## Astro releases

Reviewer: Yeah I saw that — do we generally not want to mention Starlight releases? I was suggesting including them here rather than having a whole extra section, but also OK if we’ve decided they shouldn’t be mentioned.

> ## Astro releases

Reviewer 2: That was me! I was OK mentioning releases of all kinds, BUT... then do we have to do the cloudflare adapter, the mdx integration etc?

I'm OK adding Starlight back in here, and with renaming to releases, but keeping it to Astro meant we didn't have to make those choices yet.  What do y'all think?  :sweat_smile:

- Astro 3.4: Introduction Page Partials, along with improvements in image optimization and a first preview of an experimental dev overlay. [Discover more](/blog/astro-340/).

See the `#announcements` channel in Discord for all releases across the Astro repos, including Starlight, the Astro VS Code extension, compiler, packages, and integrations! Follow [Astro on GitHub](https://github.com/withastro/) for all updates.

## Hacktoberfest

[Hacktober](https://hacktoberfest.com/) is officially over now! Thank you all for your amazing contributions and PRs! Here are some of our statistics:

- 2 Astro core PRs
- 49 Starlight PRs
- 62 Docs PRs

## Docs

We extended the tutorial with some requested features!

- [Content Collections](https://docs.astro.build/en/tutorials/add-content-collections/)
- [View Transitions](https://docs.astro.build/en/tutorials/add-view-transitions/)

"Talking and Doc'ing" is our weekly live Discord call where we involve the community as we work on docs projects! Often we jump into a shared Codespace or HackMD document and everyone can participate. This month's activities included:

- Live reviewing and testing the two new mini-tutorials

> - Improved our docs PR template and created a new GitHub action to welcome first-time contributors

Reviewer: The verb tense here is not the same as the other two bullets. Guess they should be consistent?

```suggestion
- Improving our docs PR template and creating a new GitHub action to welcome first-time contributors
```

- All hands on deck reviewing translation submissions using the Astro Docs checklist to get everyone's `hacktoberfest-accepted` PRs merged

## Community

### Community Awards September

Each month, Astro recognizes contributors from around the community, supported by [Astro's Open Collective](https://opencollective.com/astrodotbuild) funding. Join us in celebrating the following people who made Astro great this month!

> - ⭐️ [@lilnasy](https://github.com/lilnasy) - $500 - For outstanding contributions to the Astro codebase and community.
> - 🍾 [@itsmatteomanf](https://github.com/itsmatteomanf) - $250 - For initiative taken translating our docs into Italian.
> - 🎉 [@alexanderniebuhr](https://github.com/alexanderniebuhr) - $250 - For leadership in the development of the Cloudflare adapter.
> - 🥳 [@HiDeoo](https://github.com/hideoo) - $250 - For taking ownership in Starlight's maintenance, support, and new features.
> - 🎈 [@OliverSpeir](https://github.com/oliverspeir) - $250 - For a significant presence and impact in Discord support.

Reviewer: The emoji here are super cute but look a bit odd next to the sparkly bullets:

<img width="314" alt="Bulleted list with bullets like a little golden star and each line starting with an emoji" src="https://github.com/withastro/astro.build/assets/357379/c0a4d173-8116-408e-85db-ce7fc5b73525">

Maybe could move to the end of the line instead of punctuation? (It’s not docs, so maybe @sarah11918 will allow emoji punctuation 😅)

```suggestion
- [@lilnasy](https://github.com/lilnasy) - $500 - For outstanding contributions to the Astro codebase and community ⭐️
- [@itsmatteomanf](https://github.com/itsmatteomanf) - $250 - For initiative taken translating our docs into Italian 🍾
- [@alexanderniebuhr](https://github.com/alexanderniebuhr) - $250 - For leadership in the development of the Cloudflare adapter 🎉
- [@HiDeoo](https://github.com/hideoo) - $250 - For taking ownership in Starlight's maintenance, support, and new features 🥳
- [@OliverSpeir](https://github.com/oliverspeir) - $250 - For a significant presence and impact in Discord support 🎈
```

(This is also often good a11y practice because emoji names are read out by most screenreaders, so these lines could start with “star”, “champagne” etc.)

### ViteConf

This month, three Astro Core members presented talks at [ViteConf 2023](https://viteconf.org/23/).

> Learn more in our [blog post](https://astro.build/blog/viteconf-2023/) and watch [the YouTube playlist](https://www.youtube.com/watch?v=veCxKeLl35A&list=PLqGQbXn_GDmkOsHI7-Wrbv1GgAA4tJZhg) of all the talks.

Reviewer: Similar to the link a11y comment above, this one probably needs a little extra context:

```suggestion
Learn more in our [ViteConf blog post](https://astro.build/blog/viteconf-2023/) and watch [the YouTube playlist](https://www.youtube.com/watch?v=veCxKeLl35A&list=PLqGQbXn_GDmkOsHI7-Wrbv1GgAA4tJZhg) of all the talks.
```

### Content

> [Project: U2 Space Baby](https://www.chrsgrrtt.com/u2-space-baby) - Read about Chris Garrett's companion app for U2’s [residence at the Sphere](https://www.u2.com/splash) and see the [new Space Baby app for U2](https://u2spacebaby.u2.com/) app live in action!

Reviewer: Double “app”!

```suggestion
[Project: U2 Space Baby](https://www.chrsgrrtt.com/u2-space-baby) - Read about Chris Garrett's companion app for U2’s [residence at the Sphere](https://www.u2.com/splash) and see the [new Space Baby app for U2](https://u2spacebaby.u2.com/) live in action!
```

- https://astronvim.com/ by @uzaaft
- https://www.minov.studio/en/ by @suhaylmv
- https://www.linwood.dev/ by @codedoctor
- https://shamann.dev/ by @sylvhama
- https://paualbert.netlify.app/ by @akond
- https://procastinatordev.vercel.app/ by @procastinatordev
- https://www.mcdle.net/ by @elitogame
- https://www.decimal.fm/ by @chrsgrrtt
- https://dev.jeromeabel.net/ by @jeromeabel
- https://www.nunogois.com/ by @nunogois
- https://ovidius-astro-theme.netlify.app/ by @tomas_ba

### Starlight in the wild

Need inspiration for your documentation site? Here are the new Starlight sites we spotted this month!

- https://docs.corsace.io/en/
- https://certs.msfthub.wiki/
- https://aoi.js.org/
- https://www.chproxy.org/
- https://www.bunshi.org
- https://drops-of-php.hi-folks.dev/
- https://angular-challenges.vercel.app
- https://astro-docs-docs.netlify.app/
- https://docs.cataclysmbn.org/en/
- https://www.reatom.dev/
- https://zsh-test-runner-docs.vercel.app/
- https://ngxtension.netlify.app/getting-started/introduction/
- https://skyscript.js.org/
- https://nostalgist.js.org/
- https://reactnativeunistyles.vercel.app/
- https://sibiraj-s.github.io/ngx-editor

> - https://webmonetization.org/docs/

Reviewer: Including the full URLs here is pretty hard to read and also not great for screenreaders. Can we add proper link text with the title of the site/theme or at least shorter URLs so e.g. `https://ngxtension.netlify.app/getting-started/introduction/` becomes `[ngxtension.netlify.app](https://ngxtension.netlify.app/getting-started/introduction/)`? (Site title would be ideal IMO)

---
title: "What's new in Astro - October"
description: "Another busy month in Astro! From Hacktoberfest to releasing Page Partials. Read up on everything that happened this month in the Astro universe."
publishDate: "October 31, 2023"
authors:
  - elian
coverImage: "/src/content/blog/_images/whats-new-october/head.webp"
socialImage: "/src/content/blog/_images/whats-new-october/og.webp"
lang: "en"
---

## Astro releases

Astro's development keeps marching forward, with several notable releases in October:

- Astro 3.2: This update gives you more control over the browser history stack, enables page navigation with View Transitions, includes a route announcer for screen readers, and integrations can now dynamically add other integrations. [Learn more](/blog/astro-320/).
- Astro 3.3: This release introduces an experimental `<Picture />` component, enhances syntax highlighting, brings package provenance, and adds various quality-of-life improvements. [Read more](/blog/astro-330/).
- Astro 3.4: Introduction Page Partials, along with improvements in image optimization and a first preview of an experimental dev overlay. [Discover more](/blog/astro-340/).

> See the `#announcements` channel in Discord for all releases across the Astro repos, including Starlight, the Astro VS Code extension, compiler, packages and integrations! Follow [Astro on GitHub](https://github.com/withastro/) for all updates.

Reviewer 3: comma nit
```suggestion
See the `#announcements` channel in Discord for all releases across the Astro repos, including Starlight, the Astro VS Code extension, compiler, packages, and integrations! Follow [Astro on GitHub](https://github.com/withastro/) for all updates.
```

## Hacktoberfest

[Hacktober](https://hacktoberfest.com/) is officially over now! Thank you all for your amazing contributions and PRs! Here are some of our statistics:

- 2 Astro core PRs
- 49 Starlight PRs
- 62 Docs PRs

## Docs

We extended the tutorial with some requested features!

- [Content Collections](https://docs.astro.build/en/tutorials/add-content-collections/)
- [View Transitions](https://docs.astro.build/en/tutorials/add-view-transitions/)

> "Talking and Doc'ing" is our weekly live Discord call where we involve the community as we work on docs projects! Often we jump in a shared CodeSpace or HackMD document and everyone can participate. This months's activities included:

Reviewer 3: A few nits here!
- I believe the correct word here would be "into" rather than "in"
- By "CodeSpace" do you mean the GitHub product? Then it would be "Codespace" instead, so that our spelling is 1:1 with how it's written in their official copy!
- The spelling here would be "month's" rather than "months's"
```suggestion
"Talking and Doc'ing" is our weekly live Discord call where we involve the community as we work on docs projects! Often we jump into a shared Codespace or HackMD document and everyone can participate. This month's activities included:
```

- Live reviewing and testing the two new mini-tutorials
- Improved our docs PR template and created a new GitHub action to welcome first-time contributors
- All hands on deck reviewing translation submissions using the Astro Docs checklist to get everyone's `hacktoberfest-accepted` PRs merged

## Community

### Community Awards September

> Each month, Astro recognizes contributors from around the community, supported by [Astro's Open Collective](https://opencollective.com/astrodotbuild) funding. Join us in celebrating the following people that made Astro great this month!

Reviewer 3: I believe the right pronunce is "who made" vs "that made"
```suggestion
Each month, Astro recognizes contributors from around the community, supported by [Astro's Open Collective](https://opencollective.com/astrodotbuild) funding. Join us in celebrating the following people who made Astro great this month!
```

- 🍾 [@itsmatteomanf](https://github.com/itsmatteomanf) - $250 - For initiative taken translating our docs into Italian.

> - 🎉 [@alexanderniebuhr](https://github.com/alexanderniebuhr) - $250 - For leadership in development on the Cloudflare adapter.

Reviewer 3: This sentence sounds a bit hard to read and is also missing the "the" between "in" and "development". I've made a couple options you can choose from, any works for me, or you can also go with something different, the blog post is yours!
```suggestion
- 🎉 [@alexanderniebuhr](https://github.com/alexanderniebuhr) - $250 - For leading the Cloudflare adapter development.
```
```suggestion
- 🎉 [@alexanderniebuhr](https://github.com/alexanderniebuhr) - $250 - For leadership in the development of the Cloudflare adapter.
```

> - 🥳 [@HiDeoo](https://github.com/hideoo) - $250 - For taking ownership in Starlight maintenance, support and new features.

Reviewer 3: There is a similar situation here, also a couple of options to choose from!
```suggestion
- 🥳 [@HiDeoo](https://github.com/hideoo) - $250 - For owning Starlight's maintenance, support, and new features.
```

```suggestion
- 🥳 [@HiDeoo](https://github.com/hideoo) - $250 - For taking ownership in Starlight's maintenance, support, and new features.
```

- 🎈 [@OliverSpeir](https://github.com/oliverspeir) - $250 - For a significant presence and impact in Discord support.

> - 🎊 [@lilnasy](https://github.com/lilnasy) - $500 - For outstanding contributions to the Astro codebase and community.

Reviewer 3: Since Arsh is our "star contributor" this month maybe it would make sense to move him to the start of the list and change the emote to ⭐?  Not that big of a deal, but I generally expect lists with values to be in descending order, rather than crescent.

Author: Done!

## Showcase

Each week on the Astro Community Call in Discord, we shine the spotlight on projects posted by community members in our `#showcase` channel from the following week. Here's a sample of what we saw this month!

### Tools

- [Darkmatter](https://getdarkmatter.dev/) - A CMS-like interface for your Astro content collections (MacOS)
- [Frontmatter CMS](https://frontmatter.codes/) -  A Headless CMS right in your code editor - now supports Astro!
- [Search Astro Documentation Raycast Extension](https://www.raycast.com/ElianCodes/search-astro-docs) - search Astro docs directly within Raycast
- [Set Up a Static Site With Astro on Kinsta](https://kinsta.com/docs/astro-static-site/) - new guide for setting up a static site with Kinsta’s Application or Static Site hosting.

### Themes

- https://github.com/AminoffZ/catonaut by @aminoff
- https://github.com/kevinzunigacuellar/astro-supabase by @kevinzunigacuellar
- https://astro.build/themes/details/engineering-blog/ by @eddyvinck

### Websites

- https://www.noobscience.rocks/ by @noobscience
- https://fadhelmurphy.github.io/ by @mamangpro
- https://rohitk06.vercel.app/ by @rohit06

> - https://www.hvlxh.tk/ by @hitesh_dev

Reviewer 3: This one is also not a valid GitHub profile, and on Twitter this account has no posts at all, I wonder if it's also wrong? I could find the GitHub profile associated with this project is `@hvlxh`

```suggestion
- https://www.hvlxh.tk/ by @hvlxh
```

- https://astronvim.com/ by @uzaaft
- https://www.minov.studio/en/ by @suhaylmv
- https://www.linwood.dev/ by @codedoctor
- https://shamann.dev/ by @sylvhama
- https://paualbert.netlify.app/ by @akond
- https://procastinatordev.vercel.app/ by @procastinatordev
- https://www.mcdle.net/ by @elitogame
- https://www.decimal.fm/ by @chrsgrrtt
- https://dev.jeromeabel.net/ by @jeromeabel

> - https://www.nunogois.com/

Reviewer 3: This is the only website without a "by @", I wonder if this one is for GitHub profiles or Twitter/X? Anyway, here's both:

GitHub:
```suggestion
- https://www.nunogois.com/ by @nunogois
```

Twitter:
```suggestion
- https://www.nunogois.com/ by @nunogois_dev
```

> - https://ovidius-astro-theme.netlify.app/ by @tomas_ba

Reviewer 3: In GitHub's files tab any valid GitHub @ gets a link color, but not this one, which doesn't seem to be right. Could you double-check that? From what I saw the theme is under the organization `@JustGoodUI`, and the person who did the commits for it is `@astab`

Reviewer: Shouldn’t this be in the Themes section? If GH repos are important we could link to https://github.com/JustGoodUI/ovidius-astro-theme as Yan pointed out.

### Starlight in the wild

Need inspiration for your documentation site? Here are the new Starlight sites we spotted this month!

- https://docs.corsace.io/en/
- https://certs.msfthub.wiki/
- https://aoi.js.org/
- https://www.chproxy.org/
- https://www.bunshi.org
- https://drops-of-php.hi-folks.dev/
- https://angular-challenges.vercel.app
- https://astro-docs-docs.netlify.app/

> - https://twitter.com/jmeistrich/status/1710415105303982387

Reviewer 3: Feels weird this is the only case where the link to the site is actually just a Twitter link, it's a cool site but since we can't link it directly, maybe better to remove it. We have lots of sites linked here already, so that wouldn't make that much of a difference I guess? Anyway, it's up to you!

Author: Done!

- Astro 3.3: This release introduces an experimental `<Picture />` component, enhances syntax highlighting, brings package provenance, and adds various quality-of-life improvements. [Read more](/blog/astro-330/).
- Astro 3.4: Introduction Page Partials, along with improvements in image optimization and a first preview of an experimental dev overlay. [Discover more](/blog/astro-340/).

## Hacktoberfest

[Hacktober](https://hacktoberfest.com/) is officially over now! Thank you all for your amazing contributions and PRs! Here are some of our statistics:

- 2 Astro core PRs
- 49 Starlight PRs
- 62 Docs PRs

## Docs

We extended the tutorial with some requested features!

- [Content Collections](https://docs.astro.build/en/tutorials/add-content-collections/)
- [View Transitions](https://docs.astro.build/en/tutorials/add-view-transitions/)

"Talking and Doc'ing" is our weekly live Discord call where we involve the community as we work on docs projects! Often we jump in a shared CodeSpace or HackMD document and everyone can participate. This months's activities included:

- Live reviewing and testing the two new mini-tutorials

> - Reviewing brand new content for the docs

Reviewer 2: ```suggestion
```

Now that I read it again, this was super vague!  😂 let's just cut it!

- Improved our docs PR template and created a new GitHub action to welcome first-time contributors

> - Using (and improving) the Astro Docs checklist for reviewing translation PRs when you don't speak the language

Reviewer 2: ```suggestion
- All hands on deck reviewing translation submissions using the Astro Docs checklist to get everyone's `hacktoberfest-accepted` PRs merged
```
Maybe this sounds more fun?  😄

### Community Awards September

Each month, Astro recognizes contributors from around the community, supported by [Astro's Open Collective](https://opencollective.com/astrodotbuild) funding. Join us in celebrating the following people that made Astro great this month!

- 🍾 [@itsmatteomanf](https://github.com/itsmatteomanf) - $250 - For initiative taken translating our docs into Italian.
- 🎉 [@alexanderniebuhr](https://github.com/alexanderniebuhr) - $250 - For leadership in development on the Cloudflare adapter.
- 🥳 [@HiDeoo](https://github.com/hideoo) - $250 - For taking ownership in Starlight maintenance, support and new features.
- 🎈 [@OliverSpeir](https://github.com/oliverspeir) - $250 - For a significant presence and impact in Discord support.
- 🎊 [@lilnasy](https://github.com/lilnasy) - $500 - For outstanding contributions to the Astro codebase and community.

### ViteConf

This month, three Astro Core members presented talks at [ViteConf 2023](https://viteconf.org/23/).

Learn more in our [blog post](https://astro.build/blog/viteconf-2023/) and watch [the YouTube playlist](https://www.youtube.com/watch?v=veCxKeLl35A&list=PLqGQbXn_GDmkOsHI7-Wrbv1GgAA4tJZhg) of all the talks.

> 

Reviewer 2: ```suggestion

### Content

[Code Fix: The SIBA Website's Move from Create-React-App to Astro](https://brittanisavery.com/post/move-siba-to-astro) - Blog post by Brittani S. Avery
```
This is what late-minute additions were made for!

## Showcase

Each week on the Astro Community Call in Discord, we shine the spotlight on projects posted by community members in our `#showcase` channel from the following week. Here's a sample of what we saw this month!

### Tools

> - [Darkmatter](https://getdarkmatter.dev/)
> - [Frontmatter CMS](https://frontmatter.codes/)
> - [Raycast search Astro Docs](https://www.raycast.com/ElianCodes/search-astro-docs)
> - [Kinsta added support for Astro](https://kinsta.com/docs/astro-static-site/)

Reviewer 2: ```suggestion
- [Darkmatter](https://getdarkmatter.dev/) - A CMS-like interface for your Astro content collections (MacOS)
- [Frontmatter CMS](https://frontmatter.codes/) -  A Headless CMS right in your code editor - now supports Astro!
- [Search Astro Documentation Raycast Extension](https://www.raycast.com/ElianCodes/search-astro-docs) - search Astro docs directly within Raycast
- [Set Up a Static Site With Astro on Kinsta](https://kinsta.com/docs/astro-static-site/) - new guide for setting up a static site with Kinsta’s Application or Static Site hosting.
```
These could probably be beefier, so people know whether to click on them!

### Themes

- https://github.com/AminoffZ/catonaut by @aminoff
- https://github.com/kevinzunigacuellar/astro-supabase by @kevinzunigacuellar
- https://astro.build/themes/details/engineering-blog/ by @eddyvinck

### Websites

- https://www.noobscience.rocks/ by @noobscience
- https://fadhelmurphy.github.io/ by @mamangpro
- https://rohitk06.vercel.app/ by @rohit06
- https://www.hvlxh.tk/ by @hitesh_dev

> - https://astronvim.com/ by @uzaaft

Reviewer 2: This feels more like a tool, I think!

Reviewer 2: Ok gotcha! Stupid name!  👍

- https://www.minov.studio/en/ by @suhaylmv
- https://www.linwood.dev/ by @codedoctor
- https://shamann.dev/ by @sylvhama
- https://paualbert.netlify.app/ by @akond
- https://procastinatordev.vercel.app/ by @procastinatordev
- https://www.mcdle.net/ by @elitogame
- https://www.decimal.fm/ by @chrsgrrtt
- https://dev.jeromeabel.net/ by @jeromeabel
- https://www.nunogois.com/

> - https://ovidius-astro-theme.netlify.app/ by @tomas_ba

Reviewer 2: Should this be moved into themes?

---
title: "What's new in Astro - October"
description: "Another busy month in Astro! From Hacktoberfest to releasing Page Partials. Read up on everything that happened this month in the Astro universe."
publishDate: "October 31, 2023"
authors:
  - elian
coverImage: "/src/content/blog/_images/astro-3/blog-hero-post-header.webp"
socialImage: "/src/content/blog/_images/astro-3/blog-social.webp"
lang: "en"
---

> ## Astro releases

Reviewer 2: This *might* feel a little much, especially right off the top? I'm honestly not sure. We can maybe assume that people reading our blog posts have read these posts, too, so maybe this doesn't need to be as prominent? Would love other's ideas here.

- Astro `3.2`
  - Control the browser history stack
  - Trigger page navigation via JavaScript
  - Route announcer for screen readers
  - Dynamically add integrations
  - [Read more about it here](/blog/astro-320/)
- Astro `3.3`
  - experimental `<Picture />` component
  - Syntax highlighting improvements
  - Package Provenance
  - Quality of Life improvements
  - [read more about it here](/blog/astro-330/)
- Astro `3.4`
  - Page Partials
  - Image optimisation performance
  - Experimental dev overlay
  - [read more about it here](/blog/astro-340/)

## Hacktoberfest

[Hacktober](https://hacktoberfest.com/) is officially over now! Thank you all for your amazing contributions and PRs! Here are some of our statistics:

- 2 Astro core PRs
- 49 Starlight PRs
- 62 Docs PRs

## Docs

We extended the tutorial with some requested features!

- [Content Collections](https://docs.astro.build/en/tutorials/add-content-collections/)
- [View Transitions](https://docs.astro.build/en/tutorials/add-view-transitions/)

> The Astro Docs team is also working on a "Rendering Modes" page, it's still in the works and we'd love your feedback on [the open PR](https://github.com/withastro/docs/pull/4667)

Reviewer 2: ```suggestion
"Talking and Doc'ing" is our weekly live Discord call where we involve the community as we work on docs projects! Often we jump in a shared CodeSpace or HackMD document and everyone can participate. This week's activities included:

- Live reviewing and testing the two new mini-tutorials
- Reviewing brand new content for the docs
- Improved our docs PR template and created a new GitHub action to welcome first-time contributors
- Using (and improving) the Astro Docs checklist for reviewing translation PRs when you don't speak the language
```
I think T & D is maybe more interesting as being representative of "what happened" that people wouldn't otherwise know about!

## Starlight

> ### Releases & new features

Reviewer 2: Again, not sure whether this feels like a lot here. Maybe something short and snazzier?

Reviewer 2: Follow up, suggest NOT including it this month in favour of the statement above at the end of releases!

Starlight `v0.11.0` was released and brought some amazing new features, next to bugfixes.

- Component customization
- Improved MDX and sitemap integration loading
- A lot of new social icons, like, Facebook, e-mail, RSS, …
- Accessibility improvements
- Vietnamese and improved Arabic language support
- Improved `<title>` tags
- Add arbitrary HTML tags to sidebar links

You can read up on everything added to Starlight [in the release notes](https://github.com/withastro/starlight/releases).

> ### Starlight in the wild

Reviewer 2: ```suggestion
### Starlight in the wild

Need inspiration for your documentation site? Here are the new Starlight sites we spotted this week!
```
Maybe just a tiny phrase like this to introduce?

- https://docs.corsace.io/en/
- https://certs.msfthub.wiki/
- https://aoi.js.org/
- https://www.chproxy.org/
- https://www.bunshi.org
- https://drops-of-php.hi-folks.dev/
- https://angular-challenges.vercel.app
- https://astro-docs-docs.netlify.app/
- https://twitter.com/jmeistrich/status/1710415105303982387
- https://docs.cataclysmbn.org/en/
- https://www.reatom.dev/
- https://zsh-test-runner-docs.vercel.app/
- https://ngxtension.netlify.app/getting-started/introduction/
- https://skyscript.js.org/
- https://nostalgist.js.org/
- https://reactnativeunistyles.vercel.app/
- https://sibiraj-s.github.io/ngx-editor
- https://webmonetization.org/docs/

## Community

> ### Community Awards September

Reviewer 2: ```suggestion
### Community Awards September

Each month, Astro recognizes contributors from around the community, supported by [Astro's Open Collective](https://opencollective.com/astrodotbuild) funding. Join us in celebrating the following people that made Astro great this month!
```
Similarly, tiny intro sentence here!

- 🍾 [@itsmatteomanf](https://github.com/itsmatteomanf) - $250 - For initiative taken translating our docs into Italian.
- 🎉 [@alexanderniebuhr](https://github.com/alexanderniebuhr) - $250 - For leadership in development on the Cloudflare adapter.
- 🥳 [@HiDeoo](https://github.com/hideoo) - $250 - For taking ownership in Starlight maintenance, support and new features.
- 🎈 [@OliverSpeir](https://github.com/oliverspeir) - $250 - For a significant presence and impact in Discord support.
- 🎊 [@lilnasy](https://github.com/lilnasy) - $500 - For outstanding contributions to the Astro codebase and community.

### Viteconf

> This month, ViteConf took place. An amazing celebration of all things happening in the Vite world! Next to so many other great speakers, we had 3 Astro Core members giving talks. 
> 
> Read our blogpost: https://astro.build/blog/viteconf-2023/ and the [talks here](https://viteconf.org/23/replay).

Reviewer 2: ```suggestion
This month, three Astro Core members presented talks at [ViteConf 2023](https://viteconf.org/23/).

Learn more in our [blog post](https://astro.build/blog/viteconf-2023/) and watch [the YouTube playlist](https://www.youtube.com/watch?v=veCxKeLl35A&list=PLqGQbXn_GDmkOsHI7-Wrbv1GgAA4tJZhg) of all the talks.
```

> ## Showcase

Reviewer 2: ```suggestion
## Showcase

Each week on the Astro Community Call in Discord, we shine the spotlight on projects posted by community members in our `#showcase` channel from the following week. Here's a sample of what we saw this month!
```
